extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/FileService.java

```diff
@@ -1353,11 +1354,15 @@ public RepositoryFileTreeDto doGetTree( String pathId, Integer depth, String fil
     repositoryRequest.setIncludeAcls( includeAcls );
     repositoryRequest.setIncludeSystemFolders( includeSystemFolders );
 
-    RepositoryFileTreeDto tree = getRepoWs().getTreeFromRequest( repositoryRequest );
-
+    RepositoryFileTreeDto tree = null;
+    try {
+      tree = getRepoWs().getTreeFromRequest( repositoryRequest );
+    } catch ( UnifiedRepositoryException e ) {
```

Comment:
@pamval The most specific exception that I can catch here is the UnifiedRepositoryException instead of Exception.

The server console log will have the following line, with the root cause of this exception:
12:34:49,377 ERROR [FileService] org.springframework.dao.DataRetrievalFailureException: Path not found; nested exception is javax.jcr.PathNotFoundException: /pentaho/tenant0/home/admin/sched1